### ISSUE TITLE
Conflicts with x86-power-control

### ISSUE THREAD

**Issue by radix-platform** [openbmc/phosphor-state-manager#20]
Now Phosphor State Manager cannot be used with x86-power-control because service file
[phosphor-state-manager](https://github.com/openbmc/phosphor-state-manager)/[service_files](https://github.com/openbmc/phosphor-state-manager/tree/master/service_files)/xyz.openbmc_project.State.Host@.service

uses the same bus name

BusName=xyz.openbmc_project.State.Host%i

as x86-power-control service.

If system has both P-S-M and x86-power-control the boot process freezes and we have no chance to boot BMC.

Do you have any suggestions for x86 servers? If all of us have to move to P-S-M please provide solution for us. (https://github.com/openbmc/openbmc/issues/3853)

**Comment by geissonator** [comment]
I'm kind of surprised we allow both phosphor-state-manager and x86-power-control to be installed on the same system. There are interfaces, like powering on/off the system that they both implement. I think you need to choose one or the other. It seems like most x86 systems go with x86-power-control. 

PSM has a variety of different packages you can pick and choose from out at https://github.com/openbmc/openbmc/blob/master/meta-phosphor/recipes-phosphor/state/phosphor-state-manager_git.bb#L13. Maybe just bring in the specific one you want/need (if there is actually something you want/need)?

**Comment by radix-platform** [comment]
Thank you for the patient clarifications. Now I saw the difference between P-S-M and x86-power-control and also I was surprised that here is no a utility like obmcutil for x86-power-control. In P-S-M there are useful utilities and services but porting to x86-P-C is not simle. I removed unnecessary components from my image and now everything is ok. But s2600wf still not bootable (probably Intel will fix this issue soon).
